protocol-configuration/tribalchief.ts

```diff
@@ -0,0 +1,20 @@
+export default {
```

Comment:
Can you make an interface or type (preferably type) for this object? Will help devs to know what it is / how to use it.

Reply:
ok